plugins/csp-lod-bodies/src/TileSourceWebMapService.cpp

```diff
@@ -404,7 +405,7 @@ std::optional<std::string> TileSourceWebMapService::loadData(
     request.perform();
 
     auto contentType = curlpp::Info<CURLINFO_CONTENT_TYPE, std::string>::get(request);
-    fail             = contentType != "image/png" && contentType != "image/tiff";
+    fail = !cs::utils::contains(contentType, "image/png") && !cs::utils::contains(contentType, "image/tiff");
```

Comment:
I think clang-format would be happy if it was formatted like this:

```suggestion

    fail = !cs::utils::contains(contentType, "image/png") &&
           !cs::utils::contains(contentType, "image/tiff");
```